problems.sql

```diff
@@ -0,0 +1,78 @@
+-- Select titles from movies
+2.1
+SELECT title, id
+FROM movies
+Where id=6;
+2.2
+SELECT title, year
+FROM movies
+WHERE year 
+BETWEEN 2000 AND 2010
+2.3
+SELECT title, year
+FROM movies
+WHERE year NOT BETWEEN 2000 AND 2010
+2.4
+SELECT *
+FROM movies
+LIMIT 5;
+3.1
+SELECT * 
+FROM movies 
+WHERE title LIKE "Toy Story%"
+3.2
+SELECT * 
+FROM movies 
+WHERE director LIKE "John Lasseter%"
+3.3
+SELECT * 
+FROM movies 
+WHERE director Not LIKE "John Lasseter%"
+3.4
+SELECT * 
+FROM movies 
+WHERE title LIKE "Wall-_"
+4.1
+SELECT DISTINCT director 
+FROM movies 
+ORDER BY director ASC
+4.2
+SELECT * 
+FROM movies 
+ORDER BY year DESC LIMIT 4
+4.3
+SELECT * 
+FROM movies 
+ORDER BY title ASC LIMIT 5
+4.4
+SELECT * 
+FROM movies 
+ORDER BY title ASC LIMIT 5 OFFSET 5
+5.1
+SELECT * 
```

Comment:
Here, on the other hand, you only need to list just the cities and their populations. So you'd write `SELECT city, population FROM north_american_cities WHERE country = 'Canada'`